mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -888,6 +888,12 @@ class LDA @Since("1.6.0") (
   @Since("2.0.0")
   override def fit(dataset: Dataset[_]): LDAModel = {
     transformSchema(dataset.schema, logging = true)
+
+    val instr = Instrumentation.create(this, dataset)
+    instr.logParams(featuresCol, topicDistributionCol, k, maxIter, subsamplingRate,
+      checkpointInterval, keepLastCheckpoint, optimizeDocConcentration, topicConcentration,
```

Comment:
I leave it out by intention. Because `docConcentration` may be a double arrary of size `k`, it maybe too large to log.